packages/ui/package.json

```diff
@@ -84,6 +85,7 @@
     "@storybook/react": "^8.1.1",
     "@storybook/react-vite": "8.1.1",
     "@storybook/theming": "^8.1.11",
+    "@testing-library/user-event": "^14.5.2",
```

Comment:
Needed for testing clipboard functionality for the `CopyToClipboardButton` component